### DISCUSSION TITLE
Set listen port for wireguard?

### DISCUSSION THREAD

**Discussion by monkeyfella** [qdm12/gluetun#1948] [Q&A]
Hello,

How do I set listenport for wireguard? I am using torguard and they use other listen ports than the default one so I cannot connect through gluetun, I've tried WIREGUARD_LISTEN_PORT? as env but it doesn't seem to do the trick

**Comment by frepke** [comment]
I think for torguard you've to use the custom setup, more info you can find in the wiki: [https://github.com/qdm12/gluetun-wiki/blob/main/setup/providers/custom.md#wireguard](https://github.com/qdm12/gluetun-wiki/blob/main/setup/providers/custom.md#wireguard)

**Reply by monkeyfella** [reply]
hey, and thanks for replying.

I am using the custom setup, but the documentation does not cover any setting for listen port, only endpoint port..

So I am unable to connect since I don't really know how to set the listen port, I've tried the wireguard config on other computers and it works flawless

**Comment by frepke** [comment]
Maybe `- FIREWALL_VPN_INPUT_PORTS=` would do the trick.
